app/src/androidTest/java/org/mozilla/focus/helpers/TestHelper.java

```diff
@@ -3,7 +3,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-package org.mozilla.focus.activity;
+package org.mozilla.focus.helpers;
```

Comment:
\[PMD\] INFO: This class has a bunch of public methods and attributes

Reply:
This seems fine and since it's just an INFO, it doesn't cause taskCluster failures and I don't want to spend time fixing it atm – that's just what this class does.